orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -76,7 +96,7 @@ def _export_pt_1_10(g, n, *args, **kwargs):
             if call_type == "d":
                 # Got a tensor variable.
                 tensor_args.append(arg)
-                scalar_type = int(symbolic_helper.cast_pytorch_to_onnx[arg.type().scalarType()])
+                scalar_type = int(_CAST_PYTORCH_TO_ONNX[arg.type().scalarType()])
```

Comment:
This question is asked in the original PyTorch PR, but pasting here for info too.

Should we make the updated type utils `ScalarType` class public?

Reply:
Done